geowebcache/pom.xml

```diff
@@ -167,13 +167,19 @@
         <artifactId>commons-fileupload</artifactId>
         <version>${commons-fileupload.version}</version>
       </dependency>
-
-      <!-- logging dependencies -->
       <dependency>
-        <groupId>commons-logging</groupId>
-        <artifactId>commons-logging</artifactId>
-        <version>${commons-logging.version}</version>
+        <groupId>org.apache.httpcomponents</groupId>
+        <artifactId>httpclient</artifactId>
+        <version>${httpcomponents.version}</version>
+        <exclusions>
```

Comment:
Same reaction as in GEoTools... is httpclient written against commons-logging? If it is, then we'd see failures at deploy, if the expected API is not in the classpath.